### ISSUE TITLE
Page builder contents disappear when switching over to Default editor.

### ISSUE THREAD

**Issue by sunny04** [ThemeFuse/Unyson#601]
Hi,

Wordpress 4.2.2, Unyson 2.3.1

I'm having this issue that when I create any element in the Page builder and then switch over to the Default Editor everything that I had created disappears and the editor appears empty, but when I switch back to Visual Page Builder everything comes back. 

If I create some elements with the Visual Page Builder and Update the page, the elements show up on the website, but when I update the page while the Default Editor is active, and all the elements disappeared, I get nothing on the website.

So to sum up, anything that I put in the Visual Page Builder doesn't show up in the Default Editor, and anything that I put in the Default Editor (shortcodes included) doesn't show up in the Visual Page Builder.

Please let me know if this is a bug or an expected behavior as I don't have much experience with Page Builders.

Thank you and keep up the great work!

**Comment by llemurya** [comment]
@sunny04 It is a normal behavior of the page builder. If you choose the page builder mode it will render on the frontend shortocodes, if you switch to default WordPress editor it will works as sample page with content.
